docs/community_plugins.md

```diff
@@ -3,6 +3,47 @@
 Share your own plugins with everyone by adding to the list at [vatz-community-plugins](https://github.com/dsrvlabs/vatz-community-plugins) <br>
 Other contribution policies are available at [contribution guide](https://github.com/dsrvlabs/vatz/blob/main/docs/contributing.md). 
 
-<a href="https://github.com/dsrvlabs/vatz-community-plugins">
-	<img alt="Qries" src="https://github.com/dsrvlabs/vatz/assets/6308023/22c1a2cd-8abb-4ce6-813b-c173704ac80e">
-</a>
\ No newline at end of file
+---
+
+## VATZ Community Plugins
+
+`vatz-community-plugins` is a collection of unofficial plugins contributed by members of the community who wish to extend the functionality of the VATZ project with their own plugins.
+- This repository hosts all unofficial plugins for any protocol.
+- You are welcome to share your own plugins with the broader community.
+- Please note that community plugins are not managed by DSRV Labs. (Only official VATZ plugins are managed by DSRV.)
+
+> For official VATZ plugins under DSRV's management, please refer to the following repositories:
+> - [vatz-plugin-sysutil](https://github.com/dsrvlabs/vatz-plugin-sysutil)
+> - [vatz-plugin-cosmoshub](https://github.com/dsrvlabs/vatz-plugin-cosmoshub)
+
+
+## How to Register Your Plugin
+
+To contribute your plugin to this repository, please follow these steps:
+
+1. Ensure that your plugin meets the following prerequisites:
+	- The plugin repository must be public.
+	- The plugin must be registered with an official package library (e.g., [pkg.go.dev](https://pkg.go.dev/), [npmjs.com](https://www.npmjs.com/), etc.).
+
+2. Add your plugin to the "Plugin List" section, including the following details:
```

Comment:
Do we manage the list in this repo too? not community plugin repo?

Reply:
Yes, you are correct. We will be managing the list in the community plugin, and the content in docs/community_plugins.md was copied from the README.md of vatz-community-plugins. Previously, we had included screenshots directly from the README.md. As you mentioned, it would be better to reintroduce those screenshots once your review of this PR is complete. let's skip review docs/community_plugins.md for this PR, Can you review this
- https://github.com/dsrvlabs/vatz-community-plugins/pull/2